game/resource/English/ability/units/tooltip_drow_ranger_talents.txt

```diff
@@ -1,4 +1,4 @@
 //"DOTA_Tooltip_ability_special_bonus_unique_drow_ranger_1"                       "+20% Precision Aura Damage"
 //"DOTA_Tooltip_ability_special_bonus_unique_drow_ranger_2"                       "+550 Gust Distance/Knockback"
-"DOTA_Tooltip_ability_special_bonus_unique_drow_ranger_3"                       "+%value% Marksmanship Agility"
-"DOTA_Tooltip_ability_special_bonus_unique_drow_ranger_4"                       "+%value%s Gust Silence Duration"
+"DOTA_Tooltip_ability_special_bonus_unique_drow_ranger_3"                       "+40 Marksmanship Agility"
```

Comment:
this one, specifically, actually worked fine before..

Reply:
It doesn't work for allies when they click on u and for spectators too. At least I was told....

Reply:
ah, makes sense.... dota :roll_eyes: